clarifai/runners/models/method_signatures_v2.py

```diff
@@ -0,0 +1,344 @@
+import re
```

Comment:
i'm not sure I understand the difference betwee v1 and v2 without reading through everything